packages/codegen-sdk/src/parse-operation.ts

```diff
@@ -84,6 +85,22 @@ export function parseOperations(
     const model = models.find(b => b.name === fragment?.name.value);
     const modelName = model?.name ?? "UNKNOWN_MODEL";
 
+    /** Find a parent operation */
+    const parentSdkKey = sdkPath.slice(0, -1).join("_");
+    const parent = acc[parentSdkKey]?.operations.find(operation => operation.path.join("_") === sdkKey);
+
+    /** Build the path through the response to the data */
+    const responsePath = parent
+      ? printList([parent?.print.responsePath, returnedField?.name?.value], parent?.nonNull ? `.` : "?.")
+      : printList([Sdk.RESPONSE_NAME, returnedField?.name?.value], ".");
```

Comment:
Use the parent to produce a path through the response, to the data, without `?` where possible